test/e2e/components/vl-http-error-message.js

```diff
@@ -0,0 +1,15 @@
+const { VlElement } = require('vl-ui-core');
+const { By } = require('selenium-webdriver');
+
+class VlHttpErrorMessage extends VlElement {
```

Comment:
Best nog een uitbreiding voorzien om het type van foutmelding te verifiëren. Al dan niet later door @cambiph te voorzien.

Reply:
@cambiph is er nu al aan bezig